### PR TITLE
[WIP] update stampede instructions

### PR DESCRIPTION
**Pull request by tjhei**
I am playing with ASPECT on stampede and I found a couple of problems in the instructions.

### CODE REVIEW

**Comment by tjhei** [comment]
Please don't merge yet. @jdannberg please test.

File: doc/install/aspect-setup_stampede.sh [hunk 1]
```diff
@@ -4,8 +4,6 @@
 # at 2015 ASPECT Hackathon. Refer to stampede.pdf for
 # further information.
 
-set -x
-
```
Comment (jperryhouts) [inline doc/install/aspect-setup_stampede.sh]: I like the `set -x`, personally. It makes the script a bit less of a black box, and helps debug if something goes wrong.

File: doc/install/aspect-setup_stampede.sh [hunk 2]
```diff
@@ -31,12 +29,12 @@ case $1 in
 
     "trilinos")
         # Build Trilinos
-     if [ ! -f $HOME/Downloads/trilinos-11.12.1-Source.tar ]; then
-            echo "You need trilinos-11.12.1-Source.tar in ~/Downloads"
+     if [ ! -f $HOME/Downloads/trilinos-11.12.1-Source.tar.gz ]; then
+            echo "You need trilinos-11.12.1-Source.tar.gz in ~/Downloads"
             exit 1
```
Comment (dsarahstamps) [inline doc/install/aspect-setup_stampede.sh]: We started with the tar file because there were issues with uncompressing .gz using the script.

**Comment by bangerth** [comment]
What's the status of this right now?

**Comment by tjhei** [comment]
I was hoping @jdannberg would try the instructions.

**Comment by bangerth** [comment]
@jdannberg ?

**Comment by naliboff** [comment]
Revisiting these instructions right now and have used them to compile p4est, Trilinos (11.12.1) and dealii (master).  

Trilinos 12.4.2 is not building successfully with these commands and the issue appears to be interfacing to the MKL library. Someone at TACC is looking into this issue.

The development version of ASPECT is also not building and this seems to be related to hdf5 output for particles.  I'll confirm that ASPECT is compiling and running as soon as this is fixed.

**Comment by naliboff** [comment]
It turns out there is an additional compiling error (see below) that occurs for all versions of ASPECT (1.4, master, etc). This is a gcc 4.7 and c++11 compatibility issue, yes?  For reference, I am using the dealii development version and step-32 runs successfully..  

/work/01038/jboff/Software/dealii/dealii/include/deal.II/bundled/boost/serialization/map.hpp:65:51: error: ‘class std::map<std::basic_string<char>, std::basic_string<char> >’ has no member named ‘emplace_hint’

**Comment by tjhei** [comment]
> This is a gcc 4.7 and c++11 compatibility issue, yes?

Did you see https://github.com/geodynamics/aspect/pull/1265 and do you have this fix in your current clone?

**Comment by naliboff** [comment]
@tjhei - I did indeed see fix #1265, which is included in the current master clone.  Not sure if this helps narrow things down or if it is unrelated, but I do get the following warning as well:
  cc1plus: warning: unrecognized command line option "-Wno-literal-suffix" [enabled by default]

**Comment by tjhei** [comment]
Can you open an issue and include your detailed.log from your deal.II installation please?

**Comment by naliboff** [comment]
@tjhei - Sure thing, I'll open the issue later this afternoon.

**Comment by naliboff** [comment]
Before executing cmake when building deal.II, one needs to specify the following option to avoid the issue discussed above:
`export CXXFLAGS=-DBOOST_NO_CXX11_HDR_UNORDERED_MAP`

**Comment by gassmoeller** [comment]
Since #1271 is closed and Stampede1 is out of production, I close this as outdated. I will open an issue about removing the installation instructions from the repo and adding new instructions for stampede2.

**Pull request closed without merging**